package.json

```diff
@@ -62,7 +62,7 @@
     "style-loader": "0.23.1",
     "webpack": "4.28.2",
     "webpack-cli": "3.1.2",
-    "webpack-dev-server": "3.1.12"
+    "webpack-dev-server": "3.1.10"
   },
```

Comment:
I assume this is bug after merging, that you put lower version?

Reply:
https://github.com/webpack/webpack-dev-server/issues/1604

Reply:
ok